micro-demos/pods/demo.sh

```diff
@@ -28,7 +28,6 @@ run "kubectl --namespace=demos get pods"
 desc "Get the pod's IP"
 run "kubectl --namespace=demos get pod pods-demo -o yaml | grep podIP"
 
-trap "" SIGINT
```

Comment:
I don't honestly recall why I added it, but it was annoying me when running tests..